src/test/java/org/kiwiproject/dropwizard/util/health/ProcessLaunchHealthCheckTest.java

```diff
@@ -0,0 +1,86 @@
+package org.kiwiproject.dropwizard.util.health;
+
+import static org.kiwiproject.base.KiwiStrings.f;
+import static org.kiwiproject.dropwizard.util.health.ProcessLaunchHealthCheck.ECHO_MESSAGE;
+import static org.kiwiproject.test.assertj.dropwizard.metrics.HealthCheckResultAssertions.assertThatHealthCheck;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.common.base.Charsets;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.kiwiproject.base.process.ProcessHelper;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+
+@DisplayName("ProcessLaunchHealthCheck")
+class ProcessLaunchHealthCheckTest {
+
+    private ProcessLaunchHealthCheck healthCheck;
+    private ProcessHelper processes;
+    private Process process;
+
+    @BeforeEach
+    void setup() {
+        process = mock(Process.class);
+        processes = mock(ProcessHelper.class);
+        healthCheck = new ProcessLaunchHealthCheck(processes);
+    }
+
+    @Nested
+    class IsHealthy {
+
+        @Test
+        void whenEchoProcessCanBeLaunched() {
+            when(processes.launch("echo", ECHO_MESSAGE)).thenReturn(process);
```

Comment:
Update this and the others to include training newline and/or carriage return. Maybe use `@ParameterizedTest` with `@ValueSource`